src/helpers/requesters/BaseRequester.js

```diff
@@ -56,13 +56,14 @@ class BaseRequester {
       headers: headers,
       body: JSON.stringify(params),
     }).then((response) => {
+      if (!response.ok) { throw response; }
```

Comment:
does throwing `response.json()` not work here? or does it cause that nested promise problem we had?

Reply:
Yeah same promise problem as before so I have to evaluate it again in the Catch. This is the solution I dug from some guy online who was having similar issues